compiler/codegen/expr.cpp

```diff
@@ -5666,6 +5666,12 @@ GenRet NamedExpr::codegen() {
   return ret;
 }
 
+GenRet IfExpr::codegen() {
+  GenRet ret;
+  INT_FATAL(this, "IfExpr::codegen not implemented");
+  return ret;
+}
```

Comment:
For this kind of thing, I wouldn't add compiler/codegen/IfExpr.cpp. It could stay here or go in compiler/AST/IfExpr.cpp.